contracts/src/utils/cryptography/ecdsa.rs

```diff
@@ -186,7 +177,7 @@ fn encode_calldata(hash: B256, v: u8, r: B256, s: B256) -> Vec<u8> {
 /// and for v in (302): v ∈ {27, 28}.
 ///
 /// Most signatures from current libraries generate a unique signature
-/// with an s-value in the lower half order.ECDSAInvalidSignatureLength
+/// with an s-value in the lower half order.
```

Comment:
isn't here slightly different meaning?

Reply:
I see It wasn't grammatically correct before..

Reply:
`ECDSAInvalidSignatureLength` should not be placed there 😅